include/aspect/parameters.h

```diff
@@ -307,6 +307,8 @@ namespace aspect
     double                         start_time;
     double                         CFL_number;
     double                         maximum_time_step;
+    double                         reaction_time_step;
+    int                            reaction_steps_per_advection_step;
```

Comment:
I think we should make these local to the ODE integrator if we can. In ODE schemes, the time step size can often be chosen automatically. For the moment, just choose it somehow reasonable as a local variable. Or do you actually set it in input files?